cypress/elements/fileMenu/index.js

```diff
@@ -1,43 +1,31 @@
-/*  FIXME:
-    Most of the element selection in this file is quite poor because 
-    of the FileMenu being old and still uses MUI.
-    This can be refactored to use proper data-test selectors once the new
-    FileMenu component is in place.
-*/
-
 import { clickMenuBarFileButton } from '../menuBar'
 
-const deleteModalEl = '*[class^="MuiDialog-container"]'
-const fileMenuItemEl = '*[role="menuitem"]'
+const deleteModalEl = '[data-test="file-menu-delete-modal"]'
+const fileMenuItemEl = '[data-test="file-menu-container"] li'
```

Comment:
The `li` part is not ideal and breaks the pattern that's used for all other elements. I suggest skipping the `li` here by setting `const fileMenuItemEl = 'file-menu-container` and using `cy.getBySel(fileMenuItemEl).find('li')`instead.

Reply:
Removed the find for `li` as discussed in the other comment.

Reply:
Done.